lava/lava.rst

```diff
@@ -1,19 +1,32 @@
-Lava Package
-============
+Lava API reference
+==================
+
+This page gives an overview over all public lava modules, classes and functions. 
+
+Lava is divided into the sub-packages:
+
+- :ref:`The process library <lava/lava.proc:Lava process library>` containing commonly used :ref:`Processes <lava/lava.magma.core.process:lava.magma.core.process.process>` and :ref:`ProcessModels <lava/lava.magma.core.model:lava.magma.core.model.model>`.
+- :ref:`Magma <lava/lava.magma:Magma>`, containing the main components of Lava:
+  
+  - :ref:`Magma core <lava/lava.magma.core:lava.magma.core>`
+    base classes, definitions and functionality 
+  - :ref:`Magma compiler <lava/lava.magma.compiler:lava.magma.compiler>`
+    compiling and building the network and communication channels
+  - :ref:`Magma runtime <lava/lava.magma.runtime:lava.magma.runtime>`
+    providing a frontend for execution and control
+
+A fundamental description about Lava and its key componentes can be found in :ref:`Lava Architecture <lava architecture>`.
```

Comment:
The description is not fundamental, it is the concepts that are described.

"Lava's fundamental concepts and key components are described in ..."